force-app/main/default/objects/Activity/fields/InterviewFormatPick__c.field-meta.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>InterviewFormatPick__c</fullName>
```

Comment:
Are these fields on any layouts or flexipages?

Reply:
They are on a page layout

Reply:
Flexi pages and dynamic forms are the future brother :)